cmd/create/oidcprovider/cmd.go

```diff
@@ -190,7 +190,7 @@ func run(cmd *cobra.Command, argv []string) {
 		if !confirm.Prompt(true, confirmPromptMessage) {
 			os.Exit(0)
 		}
-		err = createProvider(r, oidcEndpointURL, clusterId)
+		err = createProvider(r, oidcEndpointURL, args.oidcConfigId, clusterId)
```

Comment:
is it possible args.oidcConfigId equals to ""?

Reply:
If it does, then it requires clusterId, if both are `""` - the api returns a proper bad request error

Reply:
Looking at other places where an oidc config id is needed in this command, all use `args`